css/contact.css

```diff
@@ -35,16 +36,16 @@ textarea {
 	background-color: var(--color-white);
 	border: 1px solid #6c6c6c;
 	width: 100%;
-	border-radius: 5px;
+	border-radius: var(--spacing10);
 	padding: var(--spacing10);
-	margin: 5px 0;
+	/* margin: 5px 0; */
 }
 .message {
 	min-height: 140px;
 	width: 100%;
 	border-radius: var(--spacing10);
 	border: none;
-	margin: var(--spacing10) 0;
+	/* margin: var(--spacing10) 0; */
```

Comment:
Oh it looks like you left code that was commented out - please remove it before merging

Reply:
comment removed

Reply:
marvelous